src/module/iotjs_module_adc.c

```diff
@@ -234,10 +234,10 @@ JHANDLER_FUNCTION(Unexport) {
 iotjs_jval_t InitAdc() {
   iotjs_jval_t jadc = iotjs_jval_create_object();
 
-  iotjs_jval_set_method(&jadc, "export", Export);
```

Comment:
Just one thing: you mentioned the `jval` in the name as the return value. However the return type of the `iotjs_jval_set_method` is `void`.